app/presenters/summary/components/property.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= govuk_summary_card(title:) do |card|
+      if show_actions
+        card.with_action { change_link }
+        card.with_action { remove_link }
+      end
+
+      render answers, editable: show_actions
+    end %>
```

Comment:
When you rebase new changes on main, this has been refactored to app/presenters/summary/components/base_record.rb which you should be able to use